gluten-ut/src/test/scala/org/apache/spark/sql/GlutenTestsTrait.scala

```diff
@@ -164,22 +165,32 @@ trait GlutenTestsTrait extends SparkFunSuite with ExpressionEvalHelper with Glut
     }
     val inputValues = values.map {
       _ match {
-        case utf8String: UTF8String =>
-          structFileSeq.append(StructField("s", StringType, utf8String == null))
-        case byteArr: Array[Byte] =>
-          structFileSeq.append(StructField("a", BinaryType, byteArr == null))
+        case boolean: java.lang.Boolean =>
+          structFileSeq.append(StructField("t", BooleanType, boolean == null))
```

Comment:
It may be more clearly to use the full type name here. Because it is hard to associate t to boolean, b to byte, etc.

Reply:
Actually it is used to generate Substrait function signature, and is meant to be consistent with Substrait's definition here: https://substrait.io/extensions/#function-signature-compound-names.

Reply:
I'm confused why those field names in struct should be consistent with Substrait's definitions.  The names of old StructField which the PR didn't touch are also not consistent,  which makes me more confused.

Reply:
Anyway, I renamed names according to https://substrait.io/extensions/#function-signature-compound-names.

Reply:
Sorry I mixed up this with `makeFuncName` in ConverterUtils. Maybe we can define the abbreviation for each type and use it here and there. For example,
```
final val substraitDouble = "fp64"
```
@zzcclp How do you think so?